app/views/search/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<body>
+<h1> Available on: </h1>
+      <% @resultsName.each do |r|%> 
```

Comment:
any way we could include the link as a clickable link? and maybe show the title of the show/movie that was found?